java/build-jar-multiplatform.sh

```diff
@@ -131,6 +134,20 @@ prepare_native() {
   done
 }
 
+check_native() {
```

Comment:
A clearer function name (or comment) might be helpful for this.

Reply:
renamed and added some comments